src/yunohost/__init__.py

```diff
@@ -0,0 +1,201 @@
+#! /usr/bin/python
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+import glob
+
+import moulinette
+from moulinette import m18n
+from moulinette.utils.log import configure_logging
+from moulinette.interfaces.cli import colorize, get_locale
+
+def is_installed():
+    return os.path.isfile('/etc/yunohost/installed')
+
+
+def cli(debug, quiet, output_as, timeout, args, parser):
+
+    init_logging(interface="cli", debug=debug, quiet=quiet)
+
+    # Check that YunoHost is installed
+    if not is_installed():
+        check_command_is_valid_before_postinstall(args)
+
+    ret = moulinette.cli(
+        ['yunohost'] + extensions(),
+        args,
+        output_as=output_as,
+        timeout=timeout,
+        parser_kwargs={'top_parser': parser},
+    )
+    sys.exit(ret)
+
+
+def api(debug, host, port):
+
+    init_logging(debug=debug)
+
+    def is_installed_api():
+        return {'installed': is_installed()}
+
+    # FIXME : someday, maybe find a way to disable route /postinstall if
+    # postinstall already done ...
+
+    ret = moulinette.api(
+        ['yunohost'] + extensions(),
+        host=host,
+        port=port,
+        routes={('GET', '/installed'): is_installed_api},
+        use_websocket=True
+    )
+    sys.exit(ret)
+
+
+def extensions():
+    # This is probably not used anywhere, but the actionsmap and code can be
```

Comment:
It's mentionned in YEP 4.7 https://yunohost.org/#/packaging_apps_guidelines so we need to trash this YEP if we remove this (probably) unused feature